app/models/ticket.rb

```diff
@@ -23,6 +23,14 @@ def paid?(user)
     ticket_purchases.paid.by_user(user).present?
   end
 
+  def no_of_paid(user)
```

Comment:
I would change this method to return an string with how many tickets out of the total are paid. I mean "2/3" if a user bought 3 tickets and paid 2.

And I am not sure if the name of the method is clear, what about `tickets_paid(user)`?